src/main/resources/schema/moderne-organizations.graphqls

```diff
@@ -39,6 +39,14 @@ type Organization {
     parent: Organization
 
     devCenter: DevCenter
+
+    repositories: [Repository!]!
```

Comment:
This should be paged. Then remove the stream size property change above.

Reply:
Ok, I will update to a paged setup.

Reply:
done